src/data/storageService.js

```diff
@@ -0,0 +1,65 @@
+import { S3Client, PutObjectCommand, GetObjectCommand } from '@aws-sdk/client-s3'
+
+/**
+ * The AWS keys are loaded in code here just for the 
+ * sake of dev / testing. They are currently being
+ * stored in the .env.local file which does not get 
+ * published with the repository.
+ */
+const s3Client = new S3Client({
+    region:'us-east-2',
+    credentials:{
+        accessKeyId:import.meta.env.VITE_AWS_ACCESS_KEY_ID,
```

Comment:
Is this a vite function? Where is .env.local typically stored?

Reply:
Yes, I should have added some comments to clarify those lines. Normally in node you would access environment variables as "process.env.{variablename}". However, when using Vite your environment variables are accessed from "import.meta.env.VITE_{variablename}". The file itself stored in the root folder (not /src) where you find your vite.config, package,json, etc. 

The access key id and secret are associated with an IAM "user" I created in your S3 account which I then generated the keys for. I'll send those to you since you won't be able to see the secret key in the S3 dashboard (it's shown once when you create it then hidden)

Note: Calling S3 using the environment variables to store the access keys is not the "correct" way to do it in a production environment. I only did it here for demonstration purposes. The recommended way is by using identity pools and Cognito. You can find the reference for that method [here](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/loading-browser-credentials-cognito.html).